assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -97,12 +97,13 @@ describe('Font Picker', () => {
 
     // Roboto option should be visible and have a selected checkmark
     const selectedRobotoOption = getByRole('option', {
-      // The "accessible name" is derived by concatenating the accessible names
-      // of all children,  which in this case is an SVG with aria-label="Selected"
-      // and a plain text node with the font name. Thus this works!
-      name: 'Selected Roboto',
```

Comment:
Why doesn't this work anymore?

Reply:
same reason as above with recommended being in the focus rectangle. Constriction of the `ScrollList` without alteration causes this to be `Recommended Selected Roboto` instead of `Selected Roboto`. 

Honestly, semantically the `Recently Used` & `Recommended` should probably be separate lists but didn't want to make major updates to the `ScrollList` to support this for this cosmetic PR. 

Happy to just make it work, but might have to alter `ScrollList` a bit unless you see a simpler solution.

Reply:
Yeah I see why that works. You could just use `{ name: /selected roboto/i }` which then just matches the RE and ignores extra text.

Reply:
Hm, yeah it's also a bit messy, that the headline is part of the item when navigating using screenreader - then "Recommended" will be read aloud for this item only indicating, that only the first item is actually recommended. But let's leave that for future fixing.